nests.json

```diff
@@ -416,6 +416,17 @@
       "status": "experimental",
       "category": "draw",
       "readme": "https://raw.githubusercontent.com/ismail-yilmaz/EscPainter/main/README.md"
-    }    
+    },
+    {
+      "name": "StackCtrl",
+      "packages": [
+        "StackCtrl"
+      ],
+      "description": "A simple widget stacking container for U++ that displays only one ctrl at a time.",
+      "repository": "https://github.com/ismail-yilmaz/StackCtrl.git",
+      "status": "stable",
```

Comment:
Shouldn't be it experimental at least for some time? This is new widget that was mainly used by you.

Reply:
OK. changed the category and status.